integration-tests/deployment/ccip/test_assertions.go

```diff
@@ -185,19 +193,36 @@ func ConfirmExecWithSeqNr(
 			if err != nil {
 				return fmt.Errorf("error to get source chain config : %w", err)
 			}
-			t.Logf("Waiting for ExecutionStateChanged on chain  %d from chain %d with expected sequence number %d, current onchain minSeqNr: %d",
-				dest.Selector, source.Selector, expectedSeqNr, scc.MinSeqNr)
+			executionState, err := offRamp.GetExecutionState(nil, source.Selector, expectedSeqNr)
+			if err != nil {
+				return fmt.Errorf("error to get execution state : %w", err)
+			}
+			t.Logf("Waiting for ExecutionStateChanged on chain %d (offramp %s) from chain %d with expected sequence number %d, current onchain minSeqNr: %d, execution state: %s",
+				dest.Selector, offRamp.Address().String(), source.Selector, expectedSeqNr, scc.MinSeqNr, executionStateToString(executionState))
```

Comment:
Adding these logs to aid in test debugging - in case the log fetching is (for whatever reason) not working.